### PR TITLE
added k8s yaml

### PR DESCRIPTION
**Pull request by dougsillars**
added the k8s yaml, and instructions in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ docker run -it -p 8888:8888 \
 
 > Note: New files are created inside Docker and will persist unless --rm option is used. If you'd like to also save the credentials and xRunBooks on your local machine, use Docker's -v parameter when you start the container.
 
+### Kubernetes
+The Docker image can be installed in K8s:
+
+* Deploy using the following command ([the Yaml file](installation/unskript-oss-k8s-deployment.yaml))
```
Comment (amit-chandak-unskript) [inline README.md:59]: helm chart is the standard way of deploying, we should instead build a helm chart for deploying the docker.

Comment (abhishek-unskript) [reply README.md:59]: Should this task be assigned to @jayasimha-raghavan-unskript ?

What about this PR? Should we merge it or delete it @amit-chandak-unskript ?

Comment (amit-chandak-unskript) [reply README.md:59]: Yes should be assigned to @jayasimha-raghavan-unskript . We can close this PR

**Comment by dougsillars** [comment]
closing as we are building a helm chart

**Pull request closed without merging**